src/es5-gfycat-sdk.js

```diff
@@ -328,6 +328,15 @@ GfycatSDK.prototype = {
     var query = '';
 
     if (typeof options.query === 'object' && Object.keys(options.query).length) {
+
+      // Omit null values from querystring
+      for (var key in options.query) {
+        // Using == intentionally to match null and undefined
+        if (options.query[key] == null) {
+          delete options.query[key]
```

Comment:
From my limited understanding, is that safe to delete elements from the argument passed to function?
Or maybe that could be done at the point where `options.query` values are populated?